### DISCUSSION TITLE
URGENT: TED Validation does not work

### DISCUSSION THREAD

**Discussion by JakubBalaz** [OP-TED/eForms-SDK#932] [Q&A] [answered]
Hi,

**the TED validation service does not seem to be working (we are getting: internal server error).** Also the validation via enotices2-Tool does not work and we are getting the message: Your notice is not complete. Please check for empty mandatory fields.

The problem is in production and in preview-environments. Please get back to us URGENTLY, since our contracting authorities need to publish notices today.

Thanks for your urgent help.

Best regards,
Jakub Balaz

Jakub Balaz

Jakub Balaz, MBA
Leiter e-Vergabe
 
ANKÖ Service Ges.m.b.H.
Anschützgasse 1
1150 Wien
t +43 1 333 66 66 22
[j.balaz@ankoe.at](mailto:j.balaz@ankoe.at)
[www.ankoe.at](http://www.ankoe.at/)

**Comment by bertrand-lorentz** [comment, marked as answer]
There was a problem with the validation service this morning in the Production and Preview environments. It has now been resolved, so things are back to normal.

For any problems related to ouru applications, please get in touch directly with the helpdesk [ted@publications.europa.eu](mailto:ted@publications.europa.eu)
